src/Ledger/Chain.lagda

```diff
@@ -120,24 +119,24 @@ data _⊢_⇀⦇_,NEWEPOCH⦈_ : NewEpochEnv → NewEpochState → Epoch → New
 \end{figure*}
 
 \begin{code}[hide]
-maybePurpose : DepositPurpose → (DepositPurpose × Credential) → Coin → Maybe Coin
-maybePurpose prps (prps' , _) c with prps ≟ prps'
-... | yes _ = just c
-... | no _ = nothing
-
-maybePurpose-prop : ∀ {prps} {x} {y}
-  → (m : (DepositPurpose × Credential) ⇀ Coin)
-  → (x , y) ∈ dom ((mapMaybeWithKeyᵐ (maybePurpose prps) m) ˢ)
-  → x ≡ prps
-maybePurpose-prop {prps = prps} {x} {y} _ xy∈dom with to dom∈ xy∈dom
-... | z , ∈mmwk with prps ≟ x | ∈-mapMaybeWithKey {f = maybePurpose prps} ∈mmwk
-... | yes refl | _ = refl
-
-filterPurpose : DepositPurpose → (DepositPurpose × Credential) ⇀ Coin → Credential ⇀ Coin
-filterPurpose prps m = mapKeys proj₂ (mapMaybeWithKeyᵐ (maybePurpose prps) m)
-  {λ where x∈dom y∈dom refl → cong (_, _)
-                            $ trans (maybePurpose-prop {prps = prps} m x∈dom)
-                            $ sym   (maybePurpose-prop {prps = prps} m y∈dom)}
+-- maybePurpose : DepositPurpose → (DepositPurpose × Credential) → Coin → Maybe Coin
+-- maybePurpose prps (prps' , _) c with prps ≟ prps'
+-- ... | yes _ = just c
+-- ... | no _ = nothing
+
+-- maybePurpose-prop : ∀ {prps} {x} {y}
+--   → (m : (DepositPurpose × Credential) ⇀ Coin)
+--   → (x , y) ∈ dom ((mapMaybeWithKeyᵐ (maybePurpose prps) m) ˢ)
+--   → x ≡ prps
+-- maybePurpose-prop {prps = prps} {x} {y} _ xy∈dom with to dom∈ xy∈dom
+-- ... | z , ∈mmwk with prps ≟ x | ∈-mapMaybeWithKey {f = maybePurpose prps} ∈mmwk
+-- ... | yes refl | _ = refl
+
+-- filterPurpose : DepositPurpose → (DepositPurpose × Credential) ⇀ Coin → Credential ⇀ Coin
+-- filterPurpose prps m = mapKeys proj₂ (mapMaybeWithKeyᵐ (maybePurpose prps) m)
+--   {λ where x∈dom y∈dom refl → cong (_, _)
+--                             $ trans (maybePurpose-prop {prps = prps} m x∈dom)
+--                             $ sym   (maybePurpose-prop {prps = prps} m y∈dom)}
```

Comment:
These still type-check, so if you have future use for them please keep them un-commented. Otherwise, you can delete them properly.